app/scripts/controllers/preferences.js

```diff
@@ -51,7 +51,6 @@ export default class PreferencesController {
         eth_sign: false,
       },
       useMultiAccountBalanceChecker: true,
-      hasDismissedOpenSeaToBlockaidBanner: false,
```

Comment:
Just for audit sake, I agree that no migration is needed for this since it's a single boolean rather than a large array or object.

Reply:
good shout, thanks for pointing it out